test/integration/bulk-load-test.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const fs = require('fs');
+const Connection = require('../../src/connection');
+const Request = require('../../src/request');
+const TYPES = require('../../src/data-type').typeByName;
+
+const debugMode = false;
+
+function getConfig() {
+  const config = JSON.parse(fs.readFileSync(process.env.HOME + '/.tedious/test-connection.json', 'utf8')).config;
+  config.options.tdsVersion = process.env.TEDIOUS_TDS_VERSION;
+  if (debugMode) {
+    config.options.debug = {
+      packet: true,
+      data: true,
+      payload: true,
+      token: true
+    };
+  }
+  return config;
+}
+
+exports.setUp = function(setUpDone) {
+  const connection = new Connection(getConfig());
+  connection.on('connect', (err) => {
+    if (err) {
+      setUpDone(err);
+      return;
+    }
+    this.connection = connection;
```

Comment:
I think one is ok actually, the functions are not too long. For longer functions, we should have some white space separation between logical chunks of code.